docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md

```diff
@@ -1,5 +1,18 @@
 # Adapter V4 Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 4.3.0 - Oct 29, 2022
+
+This version is for support of the .net 7 framework. See an overview of [supported frameworks here](https://docs.nunit.org/articles/vs-test-adapter/Supported-Frameworks.html).
+
+The support is due to the updated embedded NUnit.Engine version 3.15.2.
```

Comment:
Worth linking to the release notes of that? Or no?

Reply:
Done